### DISCUSSION TITLE
How to alias subcommand and option?

### DISCUSSION THREAD

**Discussion by foonathan** [CLIUtils/CLI11#837] [Q&A] [answered]
Is it possible to have a subcommand and an option do the same thing?

Consider git, all the following commands get you the version:

```
git -v
git --version
git version
```

I can of course add a `version` subcommand together with a `-v/--version` option and have them manually do the same thing, but is there a more elegant way?

**Comment by foonathan** [comment]
Nevermind, I found `.alias()` which appears to do the job. Although it's a bit weird if it is listed as `--version` under a subcommand.

**Comment by phlptp** [comment, marked as answer]
Usually what has been done in the past in cases where this is required is to add an option or flag with `-v` and `--version` and then a separate subcommand that has a callback that does the same thing as the option.    If it is complicated then the callback versions of the option/flag can be used and the same callback used in the subcommand callback.     This is a little "cleaner" than making a subcommand alias of "--version"

**Reply by foonathan** [reply]
Yeah, I guess that makes sense as well.
